### ISSUE TITLE
Since tensorflow addons is not available for windows this cannot be used. 

### ISSUE THREAD

**Issue by syedalamabbas** [LynnHo/CycleGAN-Tensorflow-Simple#15]
Is there a way to not be dependent on tensorflow-addons and still be able to run this ?

**Comment by LynnHo** [comment]
@syedalamabbas I use tensorflow-addons for Instance Normalization which is not included in tensorflow or tensorflow.keras. Currently, if you want to run the code without tf-addons, one way is to implement Instance Normalization by yourself.

**Comment by syedalamabbas** [comment]
Yes that is what I was going for.  But since that is the only place you use the addons, an easy way to circumvent is to simply use batch normalization instead and it should run then (perhaps and hopefully with some not so noticeable changes)

**Comment by iszotic** [comment]
@syedalamabbas Here's a variation that doesn't need tensorflow-addons https://github.com/iszotic/CycleGAN-Tensorflow-2, using batch normalization makes a hugeeee difference
